### PR TITLE
Make changes to home page

### PR DESCRIPTION
**Pull request by jeromehardaway**
Added text updates to index page.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -105,9 +106,9 @@ function IndexPage() {
             <div className="col-sm-12 our_mission_content text-center">
               <blockquote>
                 <p>
-                  Through training, mentorship, building our open source projects, and opportunities
-                  industry experts, we ensure that our troops gain the best knowledge possible
-                  regards to learning software skills that can get them employed.
+                  Through training, mentorship, building new features or enhancing our open-source projects, and opportunities
+                  to interact with industry experts, our goal is to ensure that our troops gain the best knowledge possible
+                  regarding to learning software skills that help them get gainfully employed in the tech sector.
```
Comment (eprislac) [inline src/pages/index.js:111]: I think this section 
> "help them get gainfully employed"

 would probably sound more natural as "obtain gainful employment".

File: src/pages/index.js [hunk 2]
```diff
@@ -84,9 +84,10 @@ function IndexPage() {
                 <h2>#VetsWhoCode</h2>
               </div>
               <p>
-                #VetsWhoCode is a veteran-led and operated 501(c)(3) charitable non-profit that
-                focuses on training veterans in programming and open source development with the
-                purpose of starting careers in the technology industry.
+                #VetsWhoCode is an inclusive, open-source, veteran-led and operated 501(c)(3) charitable public non-profit organization that
+                focuses on training veterans, active duty military, and military spouses in practical programming paradigms by using mentoring and open source development 
+                as the primary focus with the
+                goal of starting careers in the technology industry.
```
Comment (gixxerblade) [inline src/pages/index.js:90]: VetsWhoCode is an inclusive, open-source, veteran-led and operated 501(c)(3) charitable public non-profit organization. We focus on training veterans, active duty military, and military spouses in practical programming paradigms by using mentoring and open source development with the goal of starting careers in the technology industry.

File: src/pages/index.js (hunk 1 above)
Comment (gixxerblade) [reply src/pages/index.js:111]: Through training, mentorship, working with our open-source projects, and opportunities
to interact with industry experts, our goal is to ensure that our troops gain the best knowledge possible
while learning software skills that help them become gainfully employed in the technology sector.

File: src/pages/index.js [hunk 3]
```diff
@@ -155,15 +156,15 @@ function IndexPage() {
...
           </div>
           <div className="row">
             <div className="col-sm-12 cause_content text-center" style={{ marginBottom: 40 }}>
               {/* eslint-disable-next-line max-len */}
-              <h3 id="cause-title">Thank You For Working With #VetsWhoCode!</h3>
+              <h3 id="cause-title">Thank You For Supporting #VetsWhoCode!</h3>
               <hr />
               <h3 className="subtitle">
-                We Are Grateful To Have These Companies Partner With Us On Our Journey To Train
+                We Are Grateful To Have These Companies That Support Us On Our Journey To Train
```
Comment (gixxerblade) [inline src/pages/index.js:167]: Support. Remove That

File: src/pages/index.js [hunk 4]
```diff
@@ -84,9 +84,8 @@ function IndexPage() {
                 <h2>#VetsWhoCode</h2>
               </div>
               <p>
-                #VetsWhoCode is a veteran-led and operated 501(c)(3) charitable non-profit that
-                focuses on training veterans in programming and open source development with the
-                purpose of starting careers in the technology industry.
+              VetsWhoCode is an inclusive, open-source, veteran-led and operated 501(c)(3) charitable public non-profit organization.
```
Comment (gixxerblade) [inline src/pages/index.js:87]: Strike "charitable"

File: src/pages/index.js [hunk 5]
```diff
@@ -118,13 +117,13 @@ function IndexPage() {
         <div className="container">
           <div className="row">
             <div className="col-sm-6 event_content">
-              <h3 className="next-cohort">Next Cohort</h3>
+              <h3 className="next-cohort">Surprising Update</h3>
               <div className="event_excerpt">
-                <p>Countdown until our next #VetsWhoCode Cohort gets started:</p>
+                <p>Launching something new Veterans Day. Be sure to apply!</p>
```
Comment (gixxerblade) [inline src/pages/index.js:122]: Maybe think of wrapping in a conditional so it disappears after Veteran's Day? Also Veteran's Day with an apostrophe.

File: src/pages/about.js [hunk 6]
```diff
@@ -74,9 +76,9 @@ function About() {
                 <p className="story">
                   Launched in 2014, Vets Who Code is a non-profit dedicated to filling the nations
                   technical skills gap with America’s best. In order to close the digital talent
```
[the remark this answers is not in the document]
Comment (jeromehardaway) [reply src/pages/about.js:78]: @gixxerblade its an SEO play.

File: src/pages/about.js [hunk 7]
```diff
@@ -74,9 +76,9 @@ function About() {
...
-                  gap, ease career transition for military veterans, and give military spouses
-                  skills to provide stability, we connect and train veterans remotely in web
-                  development as they move to support their families. We believe that those who
+                  gap, ease career transition for military veterans and spouses
+                  skills to provide stability, we connect and train veterans remotely in various programming practices
+                   as they move to support their families. We believe that those who
```
[the remark this answers is not in the document]
Comment (jeromehardaway) [reply src/pages/about.js:81]: SEO play. Internet does not award brevity.

File: src/pages/index.js (hunk 4 above)
Comment (jeromehardaway) [reply src/pages/index.js:87]: Helps with the SEO as well. https://apps.irs.gov/pub/epostcard/dl/FinalLetter_86-2122804_VETSWHOCODEINC_05272021_00.tif

File: src/pages/index.js (hunk 5 above)
Comment (jeromehardaway) [reply src/pages/index.js:122]: I can work on that at a later date as that's not an non blocking change. Will fix the apostrophe.

File: src/pages/about.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (gixxerblade) [reply src/pages/about.js:78]: “in order to” is a conjunction which is used to join two clauses. ( one of them expresses an action, the other expresses the purpose )
How about "As a means to..."?

**Review by gixxerblade — APPROVED** [review]

**Review by gixxerblade — DISMISSED** [review]
Update snapshot

**Review by jeromehardaway — COMMENTED** [review]
Completed

**Pull request merged**